packages/react/src/forms/ControlGroup/ControlGroup.module.css

```diff
@@ -0,0 +1,65 @@
+.ControlGroup__container {
+  display: flex;
+  flex-direction: column;
+  gap: var(--base-size-8);
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-100);
+  border: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ControlGroup__label {
+  font-weight: var(--base-text-weight-semibold);
+  margin: 0;
+  padding: 0;
+  margin-bottom: var(--base-size-8);
+}
+
+.ControlGroup__caption {
+  font-size: var(--brand-text-size-100);
+  letter-spacing: var(--brand-text-letterSpacing-100);
+  line-height: var(--brand-text-lineHeight-100);
+  color: var(--brand-color-text-muted);
+}
```

Comment:
You should be able to get these styles for free by using <Text size="100" />. Can we use that here or are these styles doing something a bit different?

Reply:
Ah great, thanks. I think I just pulled these straight from [FormControl where they're hard-coded](https://github.com/primer/brand/blob/main/packages/react/src/forms/FormControl/FormControl.module.css#L71-L74).